html/Minimo/style.css

```diff
@@ -0,0 +1,258 @@
+.title--font {
+  font-family: "PlayfairDisplay";
+  font-size: 0.93rem;
+  font-weight: normal;
+}
+
+.small-text--font {
+  font-family: Ubuntu;
+  font-size: 0.45rem;
+  font-weight: 300;
+}
+
+.content-text--font {
+  font-family: "PlayfairDisplay";
+  font-size: 0.45rem;
+  font-weight: normal;
+}
+
+/* Container */
+.container {
+  padding: 0 70px;
+}
+
+/* Navbar */
+.navbar {
+  display: flex;
+  align-items: center;
+  margin-top: 35px;
+}
+
+.navbar__logo {
+  flex: 1;
+  font-family: "Inconsolata", monospace;
+  font-size: 0.93rem;
+  letter-spacing: 3px;
+  text-transform: uppercase;
+}
+
+.navbar__menu-list {
+  display: none;
+}
+
+.navbar__menu-list .menu__item {
+  color: #626262;
+  text-transform: uppercase;
+  cursor: pointer;
+}
+
+.navbar__hamburger-icon {
+  cursor: pointer;
+}
+
+/* Banner image */
+.banner {
+  width: 100%;
+  margin-top: 36px;
+}
+
+.content-container {
+  margin: 50px 0 61px 0;
+}
+
+/* Articles */
+.articles {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-row-gap: 100px;
+  margin-top: 120px;
+}
+
+.article__img {
+  width: 100%;
+}
+
+.article__type {
+  margin-top: 32px;
+  color: #b4ad9e;
+  text-transform: uppercase;
+}
+
+.article__title {
+  margin-top: 28px;
+  color: #626262;
+}
+
+.article__content {
+  margin-top: 34px;
+  color: #626262;
+  line-height: 1.71;
+}
+
+.article__comment {
+  margin-top: 52px;
+  color: #b4ad9e;
+  text-transform: uppercase;
+}
+
+/* Newsletter */
+.newsletter {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  height: 143.5px;
+  padding: 36px 0;
+  background: #f0f0f0;
+  text-align: center;
+}
+
+.newsletter__container {
+  display: flex;
+  flex-direction: column;
+}
+
+.newsletter__title {
+  margin-bottom: 40px;
+  color: #333333;
+  letter-spacing: 0.4px;
+}
+
+.newsletter__email {
+  display: flex;
+  align-items: center;
+  width: 90%;
+  margin: 0 auto;
+  padding-bottom: 7px;
+  border-bottom: solid 1px #000000;
+}
+
+.newsletter__email .email__input {
+  flex: 1;
+  background: transparent;
+  border: 0;
+  outline: none;
+}
+
+.newsletter__email .email__arrow {
+  margin-left: auto;
+  font-size: 0.6rem;
+}
+
+/* Load more button */
+.load-more {
+  display: flex;
+  justify-content: center;
+  margin-top: 82.5px;
+}
+
+.load-more__btn {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 164px;
+  height: 60px;
+  border: solid 0.5px #626262;
+  cursor: pointer;
+}
+
+.load-more__btn-text {
+  color: #626262;
+  font-family: "PlayfairDisplay";
+  font-size: 0.6rem;
+  font-weight: normal;
+}
+
+/* Footer */
+.footer {
+  padding: 20px 0;
+  background: #f0f0f0;
+}
+
+.footer-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.footer-container > span {
+  margin-bottom: 10px;
+}
+
+.footer__icon {
+  padding-left: 10px;
+  font-size: 14px;
+}
+
+/* Media queries */
+@media only screen and (min-width: 768px) {
```

Comment:
sometimes there is a reason to write media query after a class to see at place what you are changing and how it going to looks like